pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -389,8 +389,19 @@ type OIDCSettings struct {
 	ExtraScopes   string `json:"extraScopes,omitempty"`
 }
 
+// +kubebuilder:validation:Enum=metadata,recommended,minimal
```

Comment:
```suggestion
// +kubebuilder:validation:Enum=metadata;recommended;minimal
```

Reply:
Good catch, oops. Thinking about it, should this maybe include `""` in the Enum as well, so you can not set this field at all? That would make the API more backwards-compatible (if you don't set the field, you get `metadata`, which is the current behaviour).

Reply:
Or does that maybe happen already because of the `omitempty` on the field itself? Not sure how kubebuilder's validation interfaces with that.